query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -264,17 +264,20 @@ trait ExecPlan extends QueryCommand {
     */
   protected def args: String
 
+  // TODO(a_theimer): more generic way to handle lambda param with default?
   /**
     * Prints the ExecPlan and RangeVectorTransformer execution flow as a tree
     * structure, useful for debugging
     *
     * @param useNewline pass false if the result string needs to be in one line
     */
-  final def printTree(useNewline: Boolean = true, level: Int = 0): String = {
+  final def printTree(useNewline: Boolean = true,
+                      level: Int = 0,
+                      childSort: (ExecPlan) => Int = (ep) => 0): String = {
```

Comment:
I need to add documentation here for this `childSort` parameter. It's just for tests, so string comparisons between different `printTree` calls are more predictable.

Reply:
Added